advanced/Scripts/webpage.sh

```diff
@@ -519,7 +519,10 @@ Options:
     if [[ -n "${args[2]}" ]]; then
 
         # Sanitize email address in case of security issues
-        if [[ ! "${args[2]}" =~ ^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4}$  ]]; then
+        # Regex from https://stackoverflow.com/a/2138832/4065967
+        local regex
+        regex="^[a-z0-9!#\$%&'*+/=?^_\`{|}~-]+(\.[a-z0-9!#$%&'*+/=?^_\`{|}~-]+)*@([a-z0-9]([a-z0-9-]*[a-z0-9])?\.)+[a-z0-9]([a-z0-9-]*[a-z0-9])?\$"
```

Comment:
Looking at this, it seems to allow a lot of dangerous things, like `!` or `=` or even `?` and `$` or `/`. Do we really expect these characters to show up in email addresses?... Especially the exclamation mark may give rise to unpleasant behavior in `bash`!

Reply:
The regex on the SE page you linked goes down all the way into RFC 5322

However, there is also the comment:
> RFC 5322 leaves the domain name part open to implementation-specific choices that won’t work on the Internet today.

Reply:
Thinking again about it, we could maybe even keep the existing Regex and just remove the maximum length limitation of `4`? We do not enforce any restrictions on the former elements of the address, either.

Reply:
Whoops, you are right. This is why all regex I ever submit needs more pairs of eyes on it...

![image](https://user-images.githubusercontent.com/1998970/78571496-2a2a7900-781e-11ea-968e-94c3e32f0050.png)

Reply:
My proposal: Restore the previous regex and simply remove the maximum TLD length of four restriction:

```suggestion
        regex="^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}\$"
```